api/handler/v1beta1/adapter.go

```diff
@@ -61,6 +61,7 @@ func (a *adapter) FromProviderConfigProto(pc *guardianv1beta1.ProviderConfig) *d
 					ID:          roleProto.GetId(),
 					Name:        roleProto.GetName(),
 					Description: roleProto.GetDescription(),
+					Type:        roleProto.GetType(),
```

Comment:
as discussed earlier that we will not use `type` field from `role` and will create one from `permission` instead, let's not implement these changes https://github.com/goto/proton/pull/138

Reply:
Resolved: https://github.com/goto/guardian/pull/190/commits/6bd8fdffe0d3723f936bc84ddaf326d111b06d9b